### PR TITLE
feat: Custom prefixes and basic help command

### PR DESCRIPTION
**Pull request by matthe815**
Allows custom prefixes to be specified and a command to display all other commands

### CODE REVIEW

File: server/channelserver/handlers_cast_binary.go [hunk 1]
```diff
@@ -391,7 +397,7 @@ func handleMsgSysCastBinary(s *Session, p mhfpacket.MHFPacket) {
 			bf.SetLE()
 			chatMessage := &binpacket.MsgBinChat{}
 			chatMessage.Parse(bf)
-			if strings.HasPrefix(chatMessage.Message, "!") {
+			if strings.HasPrefix(chatMessage.Message, s.server.erupeConfig.CommandPrefix) {
```
Comment (sekaiwish) [inline server/channelserver/handlers_cast_binary.go:400]: The first line of `parseChatCommand` splits on index 1 so need to check for the prefix length.
```go
args := strings.Split(command[1:], " ")
```

File: config.json [hunk 2]
```diff
@@ -78,6 +79,11 @@
     "RealtimeChannelID": ""
   },
   "Commands": [
```
Comment (sekaiwish) [inline config.json:81]: Is it worth renaming this to `ChatCommands`? Not sure how ambigous `Commands` is.

Comment (matthe815) [reply config.json:81]: I don't think "commands" is overly ambiguous. It'd work fine I believe.

File: server/channelserver/handlers_cast_binary.go (hunk 1 above)
Comment (matthe815) [reply server/channelserver/handlers_cast_binary.go:403]: Good catch.

**Review by sekaiwish — APPROVED** [review]

**Pull request merged**